Kritika_Sachdeva/Calculator/styles.css

```diff
@@ -0,0 +1,122 @@
+body{
+    background-image: linear-gradient(to right, black, lightblue);
+    margin: 0;
+
+}
+
+.container{
+    background-color: rgba(0, 0, 0, 0.842);
+    height: 450px;
+    width: 330px;
+    border-radius: 8px;
+    position: relative;
+    top: 20%;
+    left: 40%;
+    box-shadow: 4px 7px gainsboro;
+}
+
+
+.scrn{
+    position: relative;
+    background-color: lightslategray;
+    height: 55px;
+    width: 266px;
+    margin: 2rem;
+    top: 25px;
+    border-radius: 4px;
+    
+}
+
+.scrn input{
+    position: relative;
+    left: 2px;
```

Comment:
for responsive designs start using relative units, px won't help you with responsiveness

Reply:
Done!